shtub.sh

```diff
@@ -47,10 +55,15 @@ _stub::data::prefix() {
 _stub::data::set() {
   local name="$1"
   local key="$2"
-  local value="$3"
-  local prefix
-  prefix=$(_stub::data::prefix "$name")
-  eval "export ${prefix}_${key}='$value'"
+  local value="${@:3}"
+  local prefix=$(_stub::data::prefix "$name" "$key")
+  local line="${prefix}=$value"
+  if [ -n "$($_stub_grep "${prefix}" .stubdata)" ]; then
```

Comment:
are these quotes correct? does `"$($_stub_grep "` get seen as quoted and `" .stubdata)"` as the other quote?